site/content/installation/installing-ngf/manifests.md

```diff
@@ -53,8 +53,22 @@ Deploying NGINX Gateway Fabric with Kubernetes manifests takes only a few steps.
 
 #### Stable release
 
+- For NGINX:
```

Comment:
I'm not certain all of these single dash items are the best formatting available.

There's a lot of instance of this: it seems like another level of headings or bolded text would work better.

I would favour headings, and at this level of nested depth it shouldn't muck with navigation or the table of contents.